libs/SmartStore/SmartStoreTests/SFSmartStoreFullTextSearchTests.h

```diff
@@ -27,5 +27,12 @@
 
 #import "SFSmartStoreTestCase.h"
 
-@interface SFSmartStoreFullTextSearchTests : SFSmartStoreTestCase 
+@interface SFSmartStoreFullTextSearchTests : SFSmartStoreTestCase
+
+#define kTestStore            @"testSmartStoreFullTextSearchStore"
```

Comment:
I wonder if all the ```#define``` should be ```static const NSString *``` instead. Minor difference, but I think we do the latter in our code usually. I don't care much either way.